### ISSUE TITLE
Request for training logs and detailed settings

### ISSUE THREAD

**Issue by BotMan-Sz** [emadeldeen24/TS-TCC#31]
Dear authors, 
Recently I have been studying your paper and code, however, even with your proceesed dataset, I can not repeat your reported results in your paper. Take the HAR dataset as example, there comes the following issues.

1. When I choose the random_init training, the results(77.9) is much better than the scores you reported in Table 2(57.89±5.13).  I don't know what random seed you used in your paper, but for 5 seed i tested, they all higher than your report.
![image](https://github.com/emadeldeen24/TS-TCC/assets/162956600/bd7a6b35-b966-4af2-bd21-7483cd8f41cb)
2. When I use only 5% of trainining data for supervised trainining, the accuracy  and macro F1(>0.89) is much higher than the reported scores Fig.2(MF1<0.55).  Below are the results for training with 5% training dataset (with batch size of 8).
![image](https://github.com/emadeldeen24/TS-TCC/assets/162956600/71a1a89d-1112-4e80-8ef3-92fc77cb55e2)
3. All thereported results are hard to reproduce.

Hope you could share me your experiment setting and training logs to remove doubts.

**Comment by BotMan-Sz** [comment]
[logs_10_12_2024_07_00_53.log](https://github.com/emadeldeen24/TS-TCC/files/14555258/logs_10_12_2024_07_00_53.log)
[run1_random_init_seed_0_classification_report.xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555260/run1_random_init_seed_0_classification_report.xlsx)
[logs_10_12_2024_07_07_03(supervised with 1% HAR dataset).log](https://github.com/emadeldeen24/TS-TCC/files/14555262/logs_10_12_2024_07_07_03.supervised.with.1.HAR.dataset.log)
[logs_10_12_2024_07_21_37(supervised with 5% HAR dataset).log](https://github.com/emadeldeen24/TS-TCC/files/14555263/logs_10_12_2024_07_21_37.supervised.with.5.HAR.dataset.log)
[run1_supervised_seed_0_classification_report(supervised with 1% HAR dataset).xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555264/run1_supervised_seed_0_classification_report.supervised.with.1.HAR.dataset.xlsx)
[run1_supervised_seed_0_classification_report(supervised with 5% HAR dataset).xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555265/run1_supervised_seed_0_classification_report.supervised.with.5.HAR.dataset.xlsx)

**Comment by BotMan-Sz** [comment]
![image](https://github.com/emadeldeen24/TS-TCC/assets/162956600/1516b5e4-0fce-4450-bb22-a5cb7eef7f41)
Add this in dataloader/dataloder.py/Load_Dataset() function to obtain certain ratio of tainining samples.

**Comment by emadeldeen24** [comment]
Hi,
thanks for experimenting with our code. Unfortunately, I don't have the logs anymore.
But I recall using the same parameters as yours, maybe the dropout was 0.5, but everything is the same. I'm not sure why you can not reproduce the results.

**Comment by SZ1995-sz** [comment]
Hi, 
Thanks for your response. As the authors, reproducing your own code should not be difficult, I hope you take a little time to recall your parameter configuration, etc., and run the code again, which is crucial for the reliability of your work.

**Comment by SZ1995-sz** [comment]
If needed, I can update my projects with logs to github, and i didn't make any changes to the core codes. It seems that even the Random Initialization and supervised results are not reliable, not to say other baselines.

**Comment by holydick99** [comment]
> logs_10_12_2024_07_00_53.log [run1_random_init_seed_0_classification_report.xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555260/run1_random_init_seed_0_classification_report.xlsx) [logs_10_12_2024_07_07_03（1% HAR 数据集监督）.log](https://github.com/emadeldeen24/TS-TCC/files/14555262/logs_10_12_2024_07_07_03.supervised.with.1.HAR.dataset.log) [logs_10_12_2024_07_21_37（5% HAR 数据集监督）.log](https://github.com/emadeldeen24/TS-TCC/files/14555263/logs_10_12_2024_07_21_37.supervised.with.5.HAR.dataset.log) [run1_supervised_seed_0_classification_report（1% HAR 数据集监督）.xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555264/run1_supervised_seed_0_classification_report.supervised.with.1.HAR.dataset.xlsx) [run1_supervised_seed_0_classification_report（5% HAR 监督）数据集）.xlsx](https://github.com/emadeldeen24/TS-TCC/files/14555265/run1_supervised_seed_0_classification_report.supervised.with.5.HAR.dataset.xlsx)

hello BotMan-Sz, I mentioned that in your log you use 5% data to do supervised training. Now I'm confuse where I can adjust the persentage of data to do self-supervised training. Looking forward your reply!
